src/lean/ffi/mod.rs

```diff
@@ -58,7 +49,7 @@ pub(super) fn drop_raw<T>(ptr: *mut T) {
 
 #[inline]
 #[allow(dead_code)]
-pub(super) fn raw_to_str<'a>(ptr: *const c_char) -> &'a str {
+pub(crate) fn raw_to_str<'a>(ptr: *const c_char) -> &'a str {
     let c_str = unsafe { CStr::from_ptr(ptr) };
     c_str.to_str().expect("Invalid UTF-8 string")
 }
```

Comment:
The fact that you've changed `raw_to_str` from `pub(super)` to `pub(crate)` makes me think that this function is now being used. In that case, it's no longer `dead_code`.